### PR TITLE
Fix typos in documentation and command example in comment in bug_report template

### PR DESCRIPTION
**Pull request by mnojek**
Related: #17727 

Just some small fixes of obvious typos found when reading documentation (some found with help of LanguageTool). Also, fixed the broken example of command in the comment inside bug_report.md template for GitHub.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Please complete the next sections or the issue will be closed.
 
 **OS**:
 
-<!-- MANDATORY! You can get it via `cat /etc/oss-release` for example -->
+<!-- MANDATORY! You can get it via `cat /etc/os-release` for example -->
```
Comment (uranusjr) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: Hmm, I think I saw another PR on this a few days ago, but can’t find it now.

File: BREEZE.rst [hunk 2]
```diff
@@ -718,13 +718,13 @@ The above will run mypy check for all files.
       </a>
     </div>
 
-If you want ever need to get a list of the files that will be checked (for troubleshooting when playing with the
-``--from-ref`` and ``--to-ref``
+If you will ever need to get a list of the files that will be checked (for troubleshooting when playing with the
+``--from-ref`` and ``--to-ref``) use these commands:
```
Comment (uranusjr) [inline BREEZE.rst]:
```suggestion
If you ever need to get a list of the files that will be checked (for troubleshooting when playing with 
``--from-ref`` and ``--to-ref``), use these commands:
```

I think this is the correct grammar?

**Review by uranusjr — DISMISSED** [review]

File: BREEZE.rst (hunk 2 above)
Comment (mnojek) [reply BREEZE.rst]: Yeah, I think it's better. Thanks!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mnojek) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: Maybe you saw my comment inside #17727 about it?

**Review by potiuk — APPROVED** [review]

File: BREEZE.rst [hunk 3]
```diff
@@ -718,13 +718,13 @@ The above will run mypy check for all files.
       </a>
     </div>
 
-If you want ever need to get a list of the files that will be checked (for troubleshooting when playing with the
-``--from-ref`` and ``--to-ref``
+If you ever need to get a list of the files that will be checked (for troubleshooting when playing with 
+``--from-ref`` and ``--to-ref``), use these commands:
```
Comment (uranusjr) [inline BREEZE.rst]:
```suggestion
If you ever need to get a list of the files that will be checked (for troubleshooting when playing with
``--from-ref`` and ``--to-ref``), use these commands:
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mnojek) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: Can I resolve this thread, @uranusjr ?

**Review by kaxil — APPROVED** [review]

**Pull request merged**